### ISSUE TITLE
Basic map returns error under ggmap_2.6.1 and ggplot2_2.2.1

### ISSUE THREAD

**Issue by stevenprimeaux** [dkahle/ggmap#165]
```
library(ggmap)

print(sessionInfo())
ggmap(get_map())
```

returns the following error:

```
Error in mapproj::mapproject(x, y, projection = coord$projection, parameters = coord$params,  : 
  object 'C_setproj' not found
```

Session information:

```
R version 3.4.1 (2017-06-30)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 7 x64 (build 7601) Service Pack 1

Matrix products: default

locale:
[1] LC_COLLATE=English_United States.1252  LC_CTYPE=English_United States.1252    LC_MONETARY=English_United States.1252
[4] LC_NUMERIC=C                           LC_TIME=English_United States.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] ggmap_2.6.1   ggplot2_2.2.1

loaded via a namespace (and not attached):
 [1] Rcpp_0.12.11      magrittr_1.5      maps_3.2.0        munsell_0.4.3     colorspace_1.3-2  geosphere_1.5-5   lattice_0.20-35  
 [8] rjson_0.2.15      jpeg_0.1-8        rlang_0.1.1       stringr_1.2.0     plyr_1.8.4        tools_3.4.1       grid_3.4.1       
[15] gtable_0.2.0      png_0.1-7         lazyeval_0.2.0    tibble_1.3.3      mapproj_1.2-5     reshape2_1.4.2    sp_1.2-5         
[22] stringi_1.1.5     compiler_3.4.1    RgoogleMaps_1.4.1 scales_0.4.1      proto_1.0.0
```

**Comment by stevenprimeaux** [comment]
Appears that issue may have been resolved in version 2.7 installed from GitHub.

**Comment by danielsjf** [comment]
This indeed solves this issue.

**Comment by stevenprimeaux** [comment]
Is the standard practice to close the issue now or wait until the version on CRAN has been updated? Please let me know.

**Comment by danielsjf** [comment]
You can choose of course, but most people/owners close it :-) btw, it's a nice practice to rename your developer versions. Now it is 2.7, but 2.6.999 has the nice advantage that it's clear that it is a developer version.

**Comment by stevenprimeaux** [comment]
Resolved in 2.7.
